packages/framework/ini-files/config-specs.ini

```
@@ -1036,6 +1036,8 @@ opt-set-cmake-var TpetraCore_CrsMatrix_2DRandomDist_MPI_4_RUN_SERIAL BOOL FORCE
 opt-set-cmake-var TpetraCore_MatrixMatrix_UnitTests_MPI_4_RUN_SERIAL BOOL FORCE : ON
 opt-set-cmake-var Zoltan2_TaskMappingTest3_MPI_1_RUN_SERIAL BOOL FORCE : ON
 opt-set-cmake-var KokkosKernels_batched_dla_cuda_MPI_1_SET_RUN_SERIAL BOOL FORCE : ON
+opt-set-cmake-var Kokkos_CoreUnitTest_CudaInterOpStreams_MPI_1_SET_RUN_SERIAL BOOL FORCE : ON
```

```suggestion
opt-set-cmake-var Kokkos_CoreUnitTest_CudaInterOpStreams_SET_RUN_SERIAL BOOL FORCE : ON
```
I don't think the MPI_1 is appended anymore to the testname

---

Done